nebula2/gclient/net/Session.py

```diff
@@ -39,24 +39,97 @@ def execute(self, stmt):
             resp = self._connection.execute(self._session_id, stmt)
             end_time = time.time()
             return ResultSet(resp,
-                             all_latency=int((end_time - start_time) * 1000000),
+                             all_latency=int(
+                                 (end_time - start_time) * 1000000),
                              timezone_offset=self._timezone_offset)
         except IOErrorException as ie:
             if ie.type == IOErrorException.E_CONNECT_BROKEN:
                 self._pool.update_servers_status()
                 if self._retry_connect:
                     if not self._reconnect():
                         logging.warning('Retry connect failed')
-                        raise IOErrorException(IOErrorException.E_ALL_BROKEN, ie.message)
+                        raise IOErrorException(
+                            IOErrorException.E_ALL_BROKEN, ie.message)
                     resp = self._connection.execute(self._session_id, stmt)
                     end_time = time.time()
                     return ResultSet(resp,
-                                     all_latency=int((end_time - start_time) * 1000000),
+                                     all_latency=int(
+                                         (end_time - start_time) * 1000000),
                                      timezone_offset=self._timezone_offset)
             raise
         except Exception:
             raise
 
+    def execute_json(self, stmt):
+        """execute statement and return the result as a JSON string
+            Date and Datetime will be returned in UTC
+            JSON struct:
+              "results": [
```

Comment:
1. how to determine if the query is successful? 
2. why the errors is in result content? how about using another key.
3. lack of error code?

Reply:
in test case, error has two keys, errorCode and errorMsg.
should enhance the comment.

Reply:
1. errorCode can be used to determine the status
2. This is to follow the neo4j JSON format
3. Good catch, errorCode is under the `errors`. I have added it back to the comment.